docs/howto/how-to-make-api-tool/api-tool-guide.md

```diff
@@ -0,0 +1,344 @@
+# Create a Tool with an API Hit
+In this guide, we use the [Alteryx Python SDK](https://pypi.org/project/ayx-python-sdk/) and [Alteryx Plugin CLI](https://pypi.org/project/ayx-plugin-cli/) to create a tool that pulls information from an API and finds the mean, min, and max values of the data.
+
+Table of Contents:
+- [Workspace Setup](#workspace-setup)
+    - [Create a Workspace](#1-create-a-workspace)
+    - [Create a Plugin](#2-create-a-plugin)
+- [Write API Request Tool](#write-api-request-tool)
+    - [Make the Request](#1-make-the-request)
+    - [Create a `pyarrow.Table`](#2-create-a-pyarrowtable)
+    - [Get the Min, Max, and Mean](#3-get-the-min-max-and-mean)
+    - [Putting It All Together](#4-putting-it-all-together)
+- [Package into a YXI](#package-into-a-yxi)
+- [Run Against Test Client](#run-against-test-client)
+- [Install and Run in Designer](#install-and-run-in-designer)
+    - [Method 1](#method-1)
+    - [Method 2](#method-2)
+## Workspace Setup
+---
+A plugin workspace is the project folder that houses all of your SDK plugins and metadata, managed by the `ayx_workspace.json` file.
+
+### 1. Create a Workspace
+The first step of plugin creation is to make a plugin workspace. To initialize the plugin workspace, we first create a new, empty directory. Then, we run the `sdk-workspace-init` command inside that directory. Fill out the prompts, which then starts the workspace initialization process.
```

Comment:
This second sentence reads a little funny to me. Can we update to:

Then, we run the `sdk-workspace-init` command inside that directory and fill out the prompts, which starts the workspace initialization process.